tests/modelbench_tests/test_benchmark_runner.py

```diff
@@ -121,6 +122,7 @@ def teardown_class(cls):
                 del ANNOTATORS._lookup[uid]
         cls._original_registered_annotators = None
 
+    # TODO: Remove? I think this any setup should be handled in conftest.
```

Comment:
Agreed.

Reply:
Ah thanks for reminding me. I just tested it and it's actually not necessary, so I removed it.